fjord/feedback/tests/test_api.py

```diff
@@ -366,6 +368,34 @@ def test_maximal(self):
             else:
                 eq_(getattr(feedback, field), data[field])
 
+    def test_invalid_unicode_url(self):
+        """Tests an API call with invalid unicode URL"""
+        data = {
+            'happy': True,
+            'description': u'Great!',
+            'category': u'ui',
+            'product': u'Firefox OS',
+            'channel': u'stable',
+            'version': u'1.1',
+            'platform': u'Firefox OS',
+            'locale': 'en-US',
+            'email': 'foo@example.com',
+            'url': 'தமிழகம்',
+            'manufacturer': 'OmniCorp',
+            'device': 'OmniCorp',
+            'country': 'US',
+            'user_agent': (
+                'Mozilla/5.0 (Mobile; rv:18.0) Gecko/18.0 Firefox/18.0'
+            ),
+            'source': 'email',
+            'campaign': 'email_test',
+        }
+        try:
+            r = self.client.post(reverse('feedback-api'), data)
+            eq_(r.status_code, 400)
+        except UnicodeEncodeError:
```

Comment:
You don't need to catch the error and assert False. Just let the exception propagate.